sahi/models/yolov8.py

```diff
@@ -27,7 +27,6 @@ def load_model(self):
 
         try:
             model = YOLO(self.model_path)
-            model.to(self.device)
```

Comment:
@edugzlez Hi, it's been a while I had concerns about this as RTDETR class should not depend on YOLOv8 for the device placement errors, can you override the method on `RTDetrDetectionModel` class instead and make this change ?

Reply:
@devrimcavusoglu I'veincluded that line and run the formatting script.

Reply:
It still seems that this line `model.to(self.device)` is removed, can u push your changes if didn't ? Model placement seems to be missing.

Reply:
Sorry, I thought you were referring to including it in `RTDetrDetectionModel.load_model`. I have included it for YOLOV8 and for RTDETR. I don't remember it well, but it seems that when I made the change the first time it failed for some reason.